### PR TITLE
Replace Semantic with Sem for easy typings

### PR DESCRIPTION
**Pull request by CaptJakk**
Because nobody has time to type that shit out :)

### CODE REVIEW

File: src/Polysemy.hs [hunk 1]
```diff
@@ -112,12 +113,13 @@ module Polysemy
...
+  )
+where
 
-import Data.Typeable
-import Polysemy.Internal
-import Polysemy.Internal.Combinators
-import Polysemy.Internal.TH.Effect
-import Polysemy.Internal.TH.Performance
-import Polysemy.Internal.Tactics
+import           Data.Typeable
```
Comment (CaptJakk) [inline src/Polysemy.hs]: looks like my brittany config murdered this. Will fix and repush

**Comment by isovector** [comment]
I figure we might as well just go all the way and `%s/Semantic/Sem`.

**Comment by CaptJakk** [comment]
I mean, I'm down, I can fix and reupload.

**Comment by isovector** [comment]
let's do it!

**Comment by CaptJakk** [comment]
Done.

**Comment by isovector** [comment]
Looks good to me. Can you follow up with `{-# DEPRECATED #-}` exports for all of the old names? Then we can release this as a minor version change.

**Comment by CaptJakk** [comment]
totally can, question is, can you mark newtype constructors as deprecated?

**Comment by isovector** [comment]
Just add `type Semantic = Sem` and `runSemantic = runSem` and deprecate
both?

**Comment by CaptJakk** [comment]
take a look here. export lists might need a little love, but otherwise I think this is close

**Comment by CaptJakk** [comment]
bump

**Comment by isovector** [comment]
Sorry -- in the middle of traveling! Hopefully I'll have some time to look at this tomorrow.

**Pull request merged**